GPU/Common/FragmentShaderGenerator.cpp

```diff
@@ -422,13 +422,14 @@ bool GenerateFragmentShader(const FShaderID &id, char *buffer, const ShaderLangu
 	if (colorWriteMask && !hasPackUnorm4x8) {
 		WRITE(p, "uint packUnorm4x8(vec4 v) {\n");
 		WRITE(p, "  v = clamp(v, 0.0, 1.0);\n");
-		WRITE(p, "  uvec4 u = uvec4(255.0 * v);\n");
+		WRITE(p, "  uvec4 u = uvec4(255 * uint(v));\n");
```

Comment:
This is not going to work. uint(v) kills the fractional part so it'll slam it to 0 or 255.

Reply:
Oops, forgot to finish that change.

-[Unknown]

Reply:
I had been thinking to convert it to a uint first at some point, but indeed it needs to be converted first.  Then I was thinking about the highp cast and got distracted by the other func, heh.

-[Unknown]

Reply:
Hehe, it's quite intricate stuff :)  I made many mistakes too before I got those Outrun reflections working...